### PR TITLE
fix: fix log flooding

### PR DESCRIPTION
**Pull request by ADD-SP**
Fixing those crazy logs.

### CODE REVIEW

**Review by ms2008 — APPROVED** [review]
LGTM

File: lib/resty/timerng/constants.lua [hunk 1]
```diff
@@ -40,8 +40,8 @@ local _M = {
     -- 1s
     SCALING_RECORD_INTERVAL = 1,
 
-    -- 60s
-    SACALING_INFO_LOG_INTERVAL = 60,
+    -- 10m
+    SACALING_INFO_LOG_INTERVAL = 600,
```
Comment (windmgc) [inline lib/resty/timerng/constants.lua:44]: Do you mean SCALING?

Comment (ADD-SP) [reply lib/resty/timerng/constants.lua]: done

File: lib/resty/timerng/init.lua [hunk 2]
```diff
@@ -387,11 +384,8 @@ function _M:named_at(name, delay, callback, ...)
     assert(type(delay) == "number", "expected `delay to be a number")
     assert(delay >= 0, "expected `delay` to be greater than or equal to 0")
 
-    if delay >= self.max_expire
-        or (delay ~= 0 and delay < self.opt.resolution)
+    if delay >= self.max_expire or (delay ~= 0 and delay < self.opt.resolution)
```
Comment (chronolaw) [inline lib/resty/timerng/init.lua:387]: I think the original multiple lines condition is easier to read.

Comment (ADD-SP) [reply lib/resty/timerng/init.lua]: done

File: lib/resty/timerng/init.lua [hunk 3]
```diff
@@ -387,11 +384,9 @@ function _M:named_at(name, delay, callback, ...)
     assert(type(delay) == "number", "expected `delay to be a number")
     assert(delay >= 0, "expected `delay` to be greater than or equal to 0")
 
-    if delay >= self.max_expire
-        or (delay ~= 0 and delay < self.opt.resolution)
+    if delay >= self.max_expire or
```
Comment (chronolaw) [inline lib/resty/timerng/init.lua:387]:
```suggestion
    if (delay >= self.max_expire) or
```
align to the next line.

File: lib/resty/timerng/init.lua [hunk 4]
```diff
@@ -410,12 +405,8 @@ function _M:named_every(name, interval, callback, ...)
     assert(type(interval) == "number", "expected `interval to be a number")
     assert(interval > 0, "expected `interval` to be greater than or equal to 0")
 
-    if interval >= self.max_expire
-        or interval < self.opt.resolution then
-        ngx_log(ngx_DEBUG,
-                "[timer-ng] fallback to ngx.timer.every [interval = ",
-                interval,
-                "]")
+    if interval >= self.max_expire or interval < self.opt.resolution
```
Comment (chronolaw) [inline lib/resty/timerng/init.lua:408]:
```suggestion
    if interval >= self.max_expire or 
       interval < self.opt.resolution
```

**Review by chronolaw — APPROVED** [review]

**Review by attenuation — APPROVED** [review]

**Pull request merged**